.gitmodules

```diff
@@ -2,3 +2,6 @@
 	path = third_party/nginx-tests
 	url = https://nginx.googlesource.com/nginx-tests
 	ignore = untracked
+[submodule "third_party/appengine-nginx-tests"]
```

Comment:
Please don't use git submodule.  We already include this in WORKSPACE dependency.